### PR TITLE
Mention unsupported features

### PR DESCRIPTION
**Pull request by atamanroman**
# Summary

- mention active and disabled features in text
- be more precise about the explicit authorisation process
- fix missing section about bank offered consent in EN
- fix status in docs

# Checklist for Definition of Done

Depending on the task some of the following jobs are not relevant. Please check the items or cross them out (`~~`text`~~`).

* [x] Code is technically reviewed by myself
* [x] Code is manually tested
* [x] Pipeline is green
* [x] Documentation is updated
* [x] Branch is rebased and therefore ready to merge
* [x] Git history is clean

### CODE REVIEW

File: docs/en/developer-portal-page.md [hunk 1]
```diff
@@ -46,40 +46,69 @@ _Figure 1.1: Components of the PSD2 Accelerator_
...
+- Payment-Types
+  - Single (sepa-credit-transfers)
+  - Future-Dated (sepa-credit-transfers)
+  - Periodic (sepa-credit-transfers)
+- Confirmation of Funds: Yes
+- Redirect-URLs
+  - PIS Redirect-URL: _https://sandbox-api.dev.adorsys.de/v1/online-banking/init/pis/:redirect-id_
+  - PIS Cancellation Redirect-URL: _https://sandbox-api.dev.adorsys.de/v1/online-banking/cancel/pis/:redirect-id_
+  - AIS Redirect-URL: _https://sandbox-api.dev.adorsys.de/v1/online-banking/init/ais/:redirect-id_
+- Unterstützte Consents
```
Comment (wandi34) [inline docs/en/developer-portal-page.md]: German translation in english doc

File: docs/de/developer-portal-page.md [hunk 2]
```diff
@@ -20,47 +20,75 @@ Zertifikat bezeichnet. Für PSD2-Zwecke wird das Zertfikat anhand des QcStatemen
...
+      - sepa-credit-transfers
+    PERIODIC:
+      - sepa-credit-transfers
+  paymentCancellationRedirectUrlExpirationTimeMs: 600000
+  availableAccountsConsentSupported: false
+  scaByOneTimeAvailableAccountsConsentRequired: true
+  psuInInitialRequestMandated: false
 ```
 
 ## Erste Schritte
```
Comment (wandi34) [inline docs/de/developer-portal-page.md:94]: Wrong margin-top in the rendered view

File: docs/en/developer-portal-page.md [hunk 3]
```diff
@@ -46,40 +46,69 @@ _Figure 1.1: Components of the PSD2 Accelerator_
...
+      - sepa-credit-transfers
+    PERIODIC:
+      - sepa-credit-transfers
+  paymentCancellationRedirectUrlExpirationTimeMs: 600000
+  availableAccountsConsentSupported: false
+  scaByOneTimeAvailableAccountsConsentRequired: true
+  psuInInitialRequestMandated: false
 ```
 
 ## Getting started
```
Comment (wandi34) [inline docs/en/developer-portal-page.md:114]: Wrong margin-top in the rendered view

File: docs/de/developer-portal-page.md [hunk 4]
```diff
@@ -20,47 +20,75 @@ Zertifikat bezeichnet. Für PSD2-Zwecke wird das Zertfikat anhand des QcStatemen
...
 Die beschriebenen Komponenten und das Zusammenspiel dieser, werden in Abbildung 1.1 dargestellt:
-displayed in Figure 1.1:
 
 ![PSD2 Accelerator](assets/accelerator.svg 'Figure 1.1: Components of the PSD2 Accelerator')
 _Figure 1.1: Components of the PSD2 Accelerator_
 
-### Aktuelle XS2A Konfiguration (Bank Profil)
+### Aktive XS2A Konfiguration (Bank Profil)
+
+- SCA-Approach: Redirect
```
Comment (wandi34) [inline docs/de/developer-portal-page.md:29]: Complete List of active configuration is not visible in the rendered view

Comment (atamanroman) [reply docs/de/developer-portal-page.md:29]: image

Looks correct, doesn't it?

File: docs/de/developer-portal-page.md (hunk 2 above)
Comment (atamanroman) [reply docs/de/developer-portal-page.md:94]: We'll have a look at it tomorrow.

File: docs/en/developer-portal-page.md (hunk 3 above)
Comment (atamanroman) [reply docs/en/developer-portal-page.md:114]: We'll have a look at it tomorrow.

File: docs/de/developer-portal-page.md (hunk 4 above)
Comment (wandi34) [reply docs/de/developer-portal-page.md:29]: Cant reproduce it again. It works

File: docs/de/developer-portal-page.md (hunk 2 above)
Comment (isabellaio) [reply docs/de/developer-portal-page.md:94]: CSS is fixed

File: docs/en/developer-portal-page.md (hunk 1 above)
Comment (wandi34) [reply docs/en/developer-portal-page.md]: Fixed

**Pull request merged**